src/overview/main.jsx

```diff
@@ -28,3 +28,5 @@ ReactDOM.render(
     </Provider>,
     document.getElementById('app')
 )
+
+export  default store;
```

Comment:
@obsidianart , I can get fix all the stuff exceptp this one , I coudn't figure out how to acess my state (Dates in this case) without exporting the state.